packages/overlay/imperative-api.md

```diff
@@ -0,0 +1,95 @@
+## Description
+
+While an `<sp-overlay>` element is the recommended entry point to the Spectrum Web Components Overlay API, you can also interact with this set of features via an imperative API, `Overlay.open`.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/overlay?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/overlay)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/overlay?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/overlay)
+
+```
+yarn add @spectrum-web-components/overlay
+```
+
+Import the `Overlay` class to leverage its capabilities within your application or custom element:
+
+```js
+import { Overlay } from '@spectrum-web-components/overlay';
+```
+
+Primarily, this class gives you access to the `open` method that will allow you to open an overlay:
+
+```js
+Overlay.open(
+    (overlayElement: HTMLElement), // the element that will be projected into the overlay, "content",
+    (options?: OverlayOptions)
+);
+```
+
+`Overlay.open()` is an asynchronous method that returns an `<sp-overlay>` element that wraps the `HTMLElement` provided as the `overlayElement`. While this process will set the `<sp-overlay>` element to `open`, consumers of this API will need to chose where to append this element to the DOM in order for the content to be made available in the browser.
+
+```js
+(async () => {
+    const content = document.querySelector('#content');
+    const options = {
+        offset: 0,
+        placement: 'bottom',
+        trigger: document.querySelector('#trigger'),
+        type: 'auto',
+    };
+    const overlay = await Overlay.open(content, options);
+    document.body.append(overlay);
+})();
+```
+
+Keep in mind that a changing DOM tree can update the relationship between existing content and the CSS that you have applied to it when appending the `<sp-overlay>` element to your existing DOM.
+
+## OverlayOptions
+
+When leveraging `Overlay.open()`, you can provide an optional second argument of `OverlayOptions`, with the following type:
+
+```ts
+type OverlayOptions = {
+    delayed?: boolean;
+    notImmediatelyClosable?: boolean;
+    offset?: number | [number, number];
+    placement?: Placement;
+    receivesFocus?: 'auto' | 'true' | 'false';
+    trigger?: HTMLElement | VirtualTrigger;
+    type?: 'modal' | 'page' | 'hint' | 'auto' | 'manual';
+};
+```
+
+### delayed
+
+An Overlay that is `delayed` will wait until a warm-up period has completed before opening. Once the warmup period has completed, all subsequent Overlays will open immediately. When no Overlays are opened, a cooldown period will begin. Once the cooldown has completed, the next Overlay to be opened will be subject to the warm-up period if provided that option.
```

Comment:
Are the warmup & cooldown periods fixed, & specified by SWC / Spectrum? If so, we should note that here.